data/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "radicle-data"
+version = "0.1.0"
+authors = ["The Radicle Team <dev@radicle.xyz>"]
```

Comment:
The license headers mention the foundation, should the info match or is there merit to have different contact points? Also totally unrelated to this change so can capture in an issue if worth discussing further.

Reply:
I... don't know. Did we change the policy at some point? Did we even _think_ about it?

My IANAL take would be: the foundation is the copyright holder by default, but the developers of radicle-link are not necessarily all affiliated with the foundation. Authors of substantial contributions do have the right to add their own name to the license header, or not mention the foundation at all.